packages/components/utils/src/truncate/truncate.ts

```diff
@@ -0,0 +1,39 @@
+export type truncateOptions = {
+  /**
+   * Number of characters to keep at the start of the string
+   */
+  start?: number;
+  /**
+   * Number of characters to keep at the end of the string
+   */
+  end?: number;
+  /**
+   * String to replace the truncated part
+   */
```

Comment:
maybe add the default value?? same for the other props

Reply:
You mean as jsdoc?

Reply:
just so it shows up on the editor as well